srecord/output/file/aomf.cc

```diff
@@ -86,7 +86,7 @@ srecord::output_file_aomf::content_record(unsigned long address,
     size_t maxlen = 4 * srecord::record::max_data_length;
     while (len > 0)
     {
-        unsigned char buffer[maxlen + 3];
+        auto buffer = new unsigned char[maxlen + 3];
```

Comment:
Happy with the fix overall, but should the new & delete move outside of the while loop?

Reply:
Yep, you're right. That could have been done in the old code already.
I just changed the line where it was and make sure, that the delete is at the end of block (to avoid accidental use afterwards).
Will fix.